durabletask/worker.py

```diff
@@ -353,44 +353,92 @@ def is_replaying(self) -> bool:
     def current_utc_datetime(self, value: datetime):
         self._current_utc_datetime = value
 
-    def create_timer(self, fire_at: Union[datetime, timedelta]) -> task.Task:
+    def create_timer(self, fire_at: Union[datetime, timedelta],
+                     retryable_task: Optional[task.RetryableTask] = None,
+                     retryable_type: Optional[str] = None) -> task.Task:
```

Comment:
Interesting, so in Python, we're allowed to override an abstract method with a concrete method that defines additional (optional) parameters?

Reply:
Yeah, it seems that is allowed in python: https://stackoverflow.com/a/50009140/18848251. But, I will change the interface signature as well, as current state looks like a bad practice. The implementation didn't complain, so I also missed it.

Reply:
But does this extra parameter even make sense to the developer? It's mostly internal, only, right? I don't have a problem with what you're doing here, but I think we probably want to not include `retryable_type` in the interface. Another option would be to just create a new method, like `create_timer_internal` which is called by `create_timer` and by your other code path.